include/staff/ticket-open.inc.php

```diff
@@ -423,7 +423,7 @@ class="<?php if ($cfg->isRichTextEnabled()) echo 'richtext';
                                 $s->getId(),
                                 $selected
                                  ? 'selected="selected"' : '',
-                                __($s->getName()));
```

Comment:
The purpose of `__()` is to translate the given string. Please see:
- https://github.com/osTicket/osTicket/blob/develop/include/class.translation.php#L1053-L1058

Reply:
This maybe works with your default statuses which are in the transaltion files. But when you add own entries in the custom list *Ticket Statuses* this don't work because of the custom translation format:

https://github.com/osTicket/osTicket/blob/develop/include/class.list.php#L1271-L1273